### ISSUE TITLE
issues for [[ auto-complete when typing Chinese

### ISSUE THREAD

**Issue by realwya** [notaapp/talk#401]
when I type Chinese characters after [[ expecting for linking pages, the auto completion doesn't work.
<img width="652" alt="image" src="https://github.com/notaapp/talk/assets/21314132/8f0e4a71-4343-4258-a86c-82cbb3ef8332">
<img width="650" alt="image" src="https://github.com/notaapp/talk/assets/21314132/d533b8c8-b200-4a5b-aa4d-737f53785b5b">
but after I input more characters and delete some, the searching/auto-completion starts to work. 
<img width="941" alt="image" src="https://github.com/notaapp/talk/assets/21314132/93d31204-332a-461b-ac95-0a78bc899c4f">

so I guess there are some issues about the timing for matching Chinese characters.

**Comment by astoilkov** [comment]
Yes, you are right. Someone reported this on Twitter a few days ago and I fixed it. The next release will include the fix.
